go/vt/vtctld/api.go

```diff
@@ -117,11 +113,7 @@ func newTabletWithStatsAndURL(t *topodatapb.Tablet, healthcheck discovery.Health
 		PrimaryTermStartTime: t.PrimaryTermStartTime,
 	}
 
-	if proxyTablets {
```

Comment:
Was removing this intended? I think it's also the cause of one of the test failures:

```
=== RUN   TestVtctldListAllTablets
[145](https://github.com/vitessio/vitess/actions/runs/3576643413/jobs/6015707221#step:12:146)
    main_test.go:237: 
[146](https://github.com/vitessio/vitess/actions/runs/3576643413/jobs/6015707221#step:12:147)
        url: http://localhost:16005/debug/health/
[147](https://github.com/vitessio/vitess/actions/runs/3576643413/jobs/6015707221#step:12:148)
        status code: 404 
[148](https://github.com/vitessio/vitess/actions/runs/3576643413/jobs/6015707221#step:12:149)
        want 200 for vtctld health check url
```

From https://github.com/vitessio/vitess/actions/runs/3576643413/jobs/6015707221

If this is supposed to be deleted, that test will need to be fixed up.

Reply:
Just an update, I added it back because the API can still use the tablet proxy.